docs/source/paper_index.md

```diff
@@ -290,7 +290,28 @@ config = GRPOConfig(
 )
 ```
 
+### Probability Smoothing Policy Optimisation
 
+**📜 Paper**: https://huggingface.co/papers/2509.21282
+
+In the paper [It's Not You, It's Clipping: A Soft Trust-Region via Probability Smoothing for LLM RL](https://huggingface.co/papers/2509.21282), the authors propose using probability smoothing to the behaviour policy instead of clipping (Probability Smoothing Policy Optimisation; PSPO), and apply this to GRPO. To use PSPO with GRPO, you can use the `GRPOTrainer` and set the `trust_region_method` to `pspo`, and set the smoothing strength using `smoothing_alpha` in the `GRPOConfig`.
```

Comment:
```suggestion
In this paper, the authors propose using probability smoothing to the behaviour policy instead of clipping (Probability Smoothing Policy Optimisation; PSPO), and apply this to GRPO. To use PSPO with GRPO, you can use the [`GRPOTrainer`] and set the `trust_region_method` to `"pspo"`, and set the smoothing strength using `smoothing_alpha` in the [`GRPOConfig`].
```